app/models/event_type.rb

```diff
@@ -22,4 +24,8 @@ class EventType < ActiveRecord::Base
   def length_step
     errors.add(:length, "must be multiple of #{LENGTH_STEP}") if length % LENGTH_STEP != 0
   end
+
+  def capitalize_color
+    self.color = color.upcase if color.present?
```

Comment:
Maybe we should introduce [normalize attributes gem](https://github.com/fnando/normalize_attributes)

^^ @hennevogel